src/locale/misc/calendar/en.yml

```diff
@@ -2,4 +2,6 @@ month: Month
 week: Week
 label: Pick an option
 prev: PREV
-next: NEXT
\ No newline at end of file
+next: NEXT
+moreEvents: |
+  {numEvents, number} more events
```

Comment:
You should use pluralization here. Currently it renders a single hidden event as _1 more events_, which is incorrect grammar. 😊 

Something like this should work:

```
{numEvents, plural, one {One more event} other {# more events}}
```